hoomd/hpmc/pytest/conftest.py

```diff
@@ -0,0 +1,1271 @@
+import pytest
+import hoomd
+import numpy as np
+
+
+@pytest.fixture(scope="function")
+def convex_polygon_integrator():
+    def make_shape():
+        return hoomd.hpmc.integrate.ConvexPolygon
+    return make_shape
+
+
+@pytest.fixture(scope="function")
+def convex_polygon_parameters():
```

Comment:
Maybe name this `convex_polygon_vertices_cpp

Reply:
Renamed similar fixtures d98589281